README.md

```diff
@@ -1,43 +1,48 @@
 # terraform-google-terraform-google-github-actions-runners
 
-This module was generated from [terraform-google-module-template](https://github.com/terraform-google-modules/terraform-google-module-template/), which by default generates a module that simply creates a GCS bucket. As the module develops, this README should be updated.
+Using these Terraform module you can quickly deploy Self Hosted Github Runners for jobs in your GitHub Actions workflows
 
-The resources/services/activations/deletions that this module will create/trigger are:
+## [Self Hosted Runners on Managed Instance Groups](modules/gh-runner-mig/README.md)
 
-- Create a GCS bucket with the provided name
+The gh-runner-mig module provisions the resources required to deploy Self Hosted Runners on GCP infrastructure using Managed Instance Groups
+This includes
```

Comment:
Is this meant to be on a newline? It's being rendered on the previous line.

Reply:
oops NL. Fixed